src/main/java/oracle/kubernetes/operator/builders/CallParams.java

```diff
@@ -0,0 +1,91 @@
+package oracle.kubernetes.operator.builders;
+
+import io.kubernetes.client.ProgressRequestBody;
+import io.kubernetes.client.ProgressResponseBody;
+
+public class CallParams {
```

Comment:
Two comments:

1) This reproduces behavior in CallBuilder.  Is your plan to refactor that as well?  (I'm okay with this if we preserve the client pooling behavior)

2) Need Javadoc for all public classes/methods

Reply:
Yes, I was planning on eventually adding a StepBuilder which would do similar things for steps, with an eye toward replacing CallBuilder altogether. I didn't initially add the client pooling behavior because the unit tests seem to invoke the client to do set up, and I wanted to use the same client in both the setup and the builder. Do you have an idea on how to do that well? Or does it not matter?